server/manager/src/main/java/org/apache/accumulo/manager/compaction/coordinator/CompactionCoordinator.java

```diff
@@ -243,6 +243,29 @@ protected void startRunningCleaner(ScheduledThreadPoolExecutor schedExecutor) {
     ThreadPools.watchNonCriticalScheduledTask(future);
   }
 
+  protected void startIdleCompactionWatcher() {
+
+    ScheduledFuture<?> future = schedExecutor.scheduleWithFixedDelay(this::idleCompactionWarning,
+        getTServerCheckInterval(), getTServerCheckInterval(), TimeUnit.MILLISECONDS);
+    ThreadPools.watchNonCriticalScheduledTask(future);
+  }
+
+  private void idleCompactionWarning() {
+
+    long now = System.currentTimeMillis();
+    Map<String,List<HostAndPort>> idleCompactors = getIdleCompactors();
+    TIME_COMPACTOR_LAST_CHECKED.forEach((groupName, lastCheckTime) -> {
+      if ((now - lastCheckTime) > getMissingCompactorWarningTime()
+          && idleCompactors.containsKey(groupName.canonical())) {
```

Comment:
Could add the check for queued and probably remove the todo.  If its configured and has nothing queued we probably still do not want to log a warning.   

```suggestion
          && jobQueues.getQueuedJobs(groupName) > 0
          && idleCompactors.containsKey(groupName.canonical())) {
```